labels/en-US.json

```diff
@@ -530,6 +531,7 @@
   "quoteHistory": "Quote History",
   "quoteName": "Quote Name",
   "quotePrint": "Print Quote",
+  "quotes": "Quotes",
```

Comment:
This should go after quoteSummary not before. Same place in both files. Use an alphasort if unsure.

Reply:
Added at the correct location